sw/airborne/firmwares/rotorcraft/navigation.h

```diff
@@ -48,10 +48,11 @@ extern uint8_t horizontal_mode;
 extern struct EnuCoor_i nav_segment_start, nav_segment_end;
 extern struct EnuCoor_i nav_circle_center;
 extern int32_t nav_circle_radius, nav_circle_qdr, nav_circle_radians;
-#define HORIZONTAL_MODE_WAYPOINT  0
```

Comment:
this means changing the meaning of the already existing modes, probably better to append the manual mode at the end.
Also needs to be updated in `server_global.ml` accordingly